.github/CODEOWNERS

```diff
@@ -0,0 +1,20 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
```

Comment:
@SeanKilleen What does this refer to?

Reply:
@CharliePoole I think you're asking about the idea of the global owner? This means that if anyone opens a PR, and it doesn't match any of the routes I've defined below, I'll be the one that gets listed as the "owner". 

"Owner" in this case being a GitHub term and not necessarily how we perceive it. Eventually, we may require a "code owner" to approve before something can be merged. This would enable us to set up scenarios in which as long as a code owner or an nunit core team member or me approved a PR, it could be merged (and possibly even auto-merged). 

Not sure where we want to go with it yet, but for now since there's no rules tied to it, all it will do is show an icon and notify people when a change is made to an area they care about / generally maintain.